spec/ddoc.dd

```diff
@@ -813,7 +813,7 @@ underscores or hyphens:
 $(P
 As with $(LINK2 #lists, lists), note that the initial `*` in the example above
 will be stripped because it is part of a documentation comment that is delimited
-with asterisks, so you need at least three subsequent asterisks.
+with asterisks - at least three subsequent asterisks are needed.
```

Comment:
The - needs to be an em dash, which is longer than a regular dash and should have no spaces between it and the words before and after. `&#8212;` or `&mdash;` should do it: `"asterisks&#8212;at"` becomes "asterisks&#8212;at". That means a macro is warranted.